### ISSUE TITLE
Namespace issues on ELI macros

### ISSUE THREAD

**Issue by jjwilke** [sstsimulator/sst-core#263]
The new ELI macros assume you are inside namespace SST.  Some of these may have already been fixed. I've variably had issues on the following types and have had to add "using" statements to work around the errors.

````
using SST::ComponentId_t;
using SST::Params;
using SST::ComponentDoc;
using SST::ElementInfoParam;
using SST::ElementInfoPort2;
using SST::ElementInfoStatistic;
using SST::SST_ELI_getMajorNumberFromVersion;
using SST::SST_ELI_getMinorNumberFromVersion;
using SST::SST_ELI_getTertiaryNumberFromVersion;
````

**Comment by bjmoor** [comment]
@jjwilke ,  Was this sufficiently fixed in PR #264, specifically 2afb82ce0d , or is there still more to do?

**Comment by jjwilke** [comment]
Good to go.
